tbot/tc/shell.py

```diff
@@ -61,13 +61,14 @@ def check_for_tool(host: linux.LinuxShell, tool: str) -> bool:
 
     :param linux.LinuxShell host: The host to ceck on.
     :param str tool: Name of the binary to check for.
+    :param bool force: Forces the check by not using the cache, default value is ``False``.
     :rtype: bool
     :returns: ``True`` if the tool was found and ``False`` otherwise.
     """
```

Comment:
Only if you want, otherwise I'll take care of it myself:  At the end of the doc-comment I usually add a line about such changes:

```python
    """
    :returns: ``True`` if the tool was found and ``False`` otherwise.

    .. versionchanged:: UNRELEASED

        Added the ``force`` parameter.
    """
```

Reply:
To late, sorry.